server/controllers/finance/reports/journal/index.js

```diff
@@ -32,15 +33,64 @@ function postingJournalExport(req, res, next) {
   });
 
   let report;
-
   try {
     report = new ReportManager(REPORT_TEMPLATE, req.session, options);
   } catch (e) {
     return next(e);
   }
 
-  const unposted = Journal.buildTransactionQuery(options, false);
-  const posted = Journal.buildTransactionQuery(options, true);
+  return Journal.journalEntryList(options)
```

Comment:
I don't Know if some think has been changed here, this function Journal.journalEntryList(options) is not recognised

Reply:
Does it work for you. you can try to export the journal as pdf and see if it is okay

Reply:
It was removed a couple days ago in https://github.com/IMA-WorldHealth/bhima-2.X/commit/67c3b7e17cc01696980fd818d8a1b67069a0a0b3#diff-e98d808c974246550596644bed19f516.